src/views/players/EditProfile/components/EditPlayerDanger.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div>
+    <h4 style="font-weight:bold">{{ $t('edit-profile:danger-zone') }}</h4>
+    <b-btn variant="danger" @click="$bvModal.show('account-delete')">Delete Account</b-btn>
+    <b-modal
+      id="account-delete"
+      hide-footer
+      size="xl"
+      centered
+      header-border-variant="primary"
+      body-class="py-0"
+    >
+      <template #modal-title>
+        <b>Are you sure you want to delete your account?</b>
+      </template>
+      <p>
+        We're sorry to hear that you're leaving us! Something wrong that we can help with?
+        Please get in touch at <a href="mailto:support@eternagame.org">support@eternagame.org</a>.
+      </p>
+      <p>
+        Your account will be deactivated and scheduled to be deleted in 30 days.
+        Per our privacy policy found at <a href="https://eternagame.org/terms">https://eternagame.org/terms</a>,
+        your identifying information will be removed and your contributions will be anonymized
+        (though we cannot retroactively remove names that may have been included in
+        scientific publications). If you have used our forum, you will also need to separately
+        delete your forum account at <a href="https://forum.eternagame.org/">https://forum.eternagame.org/</a>.
+      </p>
+      <p>
+        Thank you for joining us on our mission to advance science! To confirm you want to delete
+        your account, please enter your username and password below and click delete.
+      </p>
+      <b-form>
+        <b-input
+          type="text"
+          placeholder="Username"
+          v-model="username"
+          required
+        />
+        <b-input
+          type="password"
+          placeholder="Password"
+          v-model="password"
+          required
+        />
+        <transition name="fade">
+          <b-alert class="mt-3" show variant="danger" v-if="errorMessage">
+            {{ errorMessage }}
+          </b-alert>
+        </transition>
+        <b-btn variant="danger" @click="deleteAccount()" :disabled="loading">
+          Delete Account
+          <b-spinner v-if="loading" small />
+        </b-btn>
```

Comment:
Should the modal also have a cancel button?

Reply:
Definitely!